components/dmx512/dmx512.h

```diff
@@ -74,7 +76,9 @@ class DMX512Output : public output::FloatOutput, public Component {
 public:
   void set_universe(DMX512 *universe) { this->universe_ = universe; }
   void set_channel(uint16_t channel);
+  float get_state();
   void write_state(float state) override;
+  float state;
```

Comment:
This variable should be named `state_` to be consistent with the rest of the code. But what's the purpose of this variable anyway? It's overwritten in the getter and overwritten in the setter, so it could be dropped, right?

Reply:
This is in order to be able to reference the current state of the output in lambdas without a "call" pretty much like one would access the `state` variables from ESPHome's sensors.